keywords/couchbaseserver.py

```diff
@@ -247,6 +207,50 @@ def wait_for_ready_state(self):
             # All nodes are heathy if it made it to here
             break
 
+    def _create_internal_rbac_bucket_user(self, bucketname):
+        # Create user with username=bucketname and assign role
+        # bucket_admin and cluster_admin
+        roles = "cluster_admin,bucket_admin[{}]".format(bucketname)
+        password = 'password'
+
+        data_user_params = {
+            "name": bucketname,
+            "roles": roles,
+            "password": password
+        }
+
+        log_info("Creating RBAC user {} with password {} and roles {}".format(bucketname, password, roles))
+
+        rbac_url = "{}/settings/rbac/users/local/{}".format(self.url, bucketname)
+
+        resp = ""
+        try:
+            resp = self._session.put(rbac_url, data=data_user_params, auth=('Administrator', 'password'))
+            log_r(resp)
+            resp.raise_for_status()
+        except HTTPError as h:
+            log_info("resp code: {}; error: {}".format(resp, h))
+            raise RBACUserCreationError(h)
+
+    def _delete_internal_rbac_bucket_user(self, bucketname):
```

Comment:
pylint is complaining that this method name is invalid, may be because it starts with _ . Not a big deal, just letting you know.